ppa/local_settings.py.sample

```diff
@@ -113,3 +113,7 @@ LOGGING = {
 
     }
 }
+
+# https://github.com/mozilla/django-csp
+# URL to POST reports of content security policy violations to
```

Comment:
in dev, `REPORT_ONLY` defaults to `False` and `REPORT_URL` defaults to `None` (don't report), which means violations will be blocked but not reported - I think that's the configuration we want, so theoretically you shouldn't have to change anything.

maybe the simplest thing is to note that in a comment here, and then comment out the `REPORT_ONLY` setting in case you wanted to allow violations temporarily in dev to test something? `REPORT_URL` doesn't really need to be in the sample at all, as it'll only be set in qa/prod.